docs/api/networkinterfaces/MeshInterface.md

```diff
@@ -43,7 +43,7 @@ See the example application [mbed-os-example-mesh-minimal](https://github.com/AR
 
 The application below demonstrates a simple light control application, where devices can control the LED status of all devices in the network. You can build the application for the unsecure 6LoWPAN-ND or Thread network.
 
-[![View code](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-example-mesh-minimal)](https://github.com/ARMmbed/mbed-os-example-mesh-minimal/blob/mbed-os-5.10.0/main.cpp)
+[![View code](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-example-mesh-minimal)](https://github.com/ARMmbed/mbed-os-example-mesh-minimal/blob/mbed-os-5.10.0/mesh_led_control_example.cpp)
 
```

Comment:
`mesh_led_control_example.cpp` is used from` main.cpp`. I think we need both files for this example.

Reply:
Each transcluded example shows one file. Users can get to the other files by clicking the blue text at the top of the rendered example. If that's not enough, we can add an example, so there are two.

Reply:
OK, no need to add another example. I'm fine with this change.